src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/GoogleServicesDAO.scala

```diff
@@ -39,7 +39,7 @@ trait GoogleServicesDAO {
       case gjre: GoogleJsonResponseException =>
         val statusCode =
           if ( gjre.getDetails == null ) None
```

Comment:
This check should be removed because it's possible to have a code but no details.